AGENTS.md

```diff
@@ -0,0 +1,197 @@
+# AGENTS.md
+
+## Setup
+
+### Requirements
+
+- Node.js >=20.0.0
+- Python 3
+- Git (for development)
+
+### Environment
+
+- Docker (required for E2E tests via CHT Docker Helper)
+- Internet connection (for downloading dependencies and connecting to CHT instances)
+
+### Installation
+
+```bash
+# Install cht-conf globally
+npm install -g cht-conf
+
+# Install required Python dependency
+pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
+
+# For local development
+git clone https://github.com/medic/cht-conf.git
+cd cht-conf
+npm install
+```
+
+### Docker Alternative
+
+```bash
+# Run using Docker (example: run tests)
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide npm test
+# Or run end-to-end tests
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide npm run test-e2e
+# For more available commands, see the documentation at https://github.com/medic/cht-conf#usage or run:
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide --help
+```
+
+## Build & Test
+
+```bash
+# Code Quality
+npm run eslint                # Run linting checks
+
+# Testing
+npm test                      # Run unit tests with Mocha and NYC coverage
+npm run test-e2e              # Run end-to-end tests (requires Docker)
+
+# Build and Maintenance
+npm run clean                 # Clean build directory
+npm run semantic-release      # Release new version (for maintainers)
```

Comment:
No need to have this here.  We don't want AI agents trying to release a new version... 😅 

```suggestion
```